src/components/OrderedList/OrderedList.stories.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @file Ordered list stories.
+ * @copyright IBM Security 2019
+ */
+ 
+ import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { components } from '../../../.storybook';
+
+import OrderedList from '../OrderedList';
+import { ListItem, OrderedList } from '../../';
```

Comment:
- [ ] No need for a trailing slash here! 👋

```suggestion
import { ListItem, OrderedList } from '../..';
```